scss/tests/mixins/_utilities-map.test.scss

```diff
@@ -0,0 +1,339 @@
+@mixin before {
+  $utilities: (
+    "flex": (
+      responsive: true,
+      property: flex,
+      values: (fill: 1 1 auto)
+    ),
+    "align": (
+      property: vertical-align,
+      class: align,
+      values:  baseline top
+    ),
+    "margin": (
+      property: margin,
+      class: margin,
+      values: auto
+    )
+  ) !global;
+}
+
+@import "../../mixins/utilities-map";
+
+@include describe("utilities") {
+
+  @include describe("utilities-get-options") {
```

Comment:
Do we need to test `map-from` function ?

Reply:
I was considering that it could be skipped, given `utility-get-value` was tested thoroughly. Similarly, there's no test for `utility-get-values` as `utility-get-value` runs it under the hood as well. I think the current tests cover all (or at least most to feel safe,  shame there's no way to gather code coverage) of the code in `_utilities-map.scss`. We can always add new tests if we ever refactor and have "utilities-get-values" or "map-from" not called anymore, I think.